### DISCUSSION TITLE
How to scroll with trackpad when using add-on?

### DISCUSSION THREAD

**Discussion by kevdogg** [hassio-addons/addon-ssh#746] [Q&A]
I accessing HAOS via use macbook using the advanced ssh and terminal add-on.  The trackpad simply "scrolls" the command history and not the screen.  Is there a way to enable screen scrolling and not command history scrolling?

**Comment by wise-rice** [comment]
+1

**Comment by tms320** [comment]
+1
Can't scroll terminal output buffer via mouse scroll wheel.
